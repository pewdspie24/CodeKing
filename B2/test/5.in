10
11210511911511111997122107
99116114108105121111
118112117120118108101114
115110
101
99104115110
105104116101102119100121117105
107101113
10911611597104119
112
