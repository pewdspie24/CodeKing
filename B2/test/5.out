piwsowazk
ctrliyo
vpuxvler
sn
e
chsn
ihtefwdyui
keq
mtsahw
p
